### PR TITLE
Feature | Add profile view

### PR DESCRIPTION
**Pull request by EdgarB**
## Description
[Ticket](https://github.com/orgs/TelosLabs/projects/5/views/1?pane=issue&itemId=68736367)

## How has this been tested?

Please mark the tests that you ran to verify your changes. If difficult to test, consider providing instructions so reviewers can test.

- [ ] Manual testing
- [ ] System tests
- [ ] Unit tests
- [ ] None

## Checklist

- [ ] CI pipeline is passing
- [ ] My code follows the conventions of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (if applicable)
- [ ] I have added seed data to the database (if applicable)

## Release tasks

Add any tasks that need to be done before/after the release of this feature.

## Screenshots/Loom

This section is relevant in case we want to share progress with the team, otherwise, it can be omitted.

### CODE REVIEW

File: app/views/profiles/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<div class="flex flex-col items-center bg-gray-8 relative min-h-screen justify-between ">
```
[the remark this answers is not in the document]
Comment (EdgarB) [reply app/views/profiles/show.html.erb:1]: For now, I'll keep the social media buttons hidden if they weren't added.

File: app/views/profiles/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    <div class="flex flex-col items-center bg-cover bg-bottom bg-no-repeat max-h-[430px] w-full pt-6 px-5 pb-12 max-w-screen-sm" style="background-image: url(<%= asset_path 'background_design.svg' %>)">
+        <div class="flex flex-col items-start z-10">
+            <div class="flex flex-row items-center">
+                <% if @profile.image.attached? %>
```
[the remark this answers is not in the document]
Comment (EdgarB) [reply app/views/profiles/show.html.erb:5]: This is already solve on the navigation PR.

**Comment by Sergio-e** [comment]
I just noticed that we forgot to add a way to allow users to remove an image if they already uploaded one. It would be nice if you and Charlie could come up with a simple way to solve it
image

**Comment by Sergio-e** [comment]
I'm unable to remove images. I'm seeing the following after clicking on the "-" icon
image

**Comment by Sergio-e** [comment]
Seems like the icon should be on the upper right corner instead of on the lower left
image

**Comment by Sergio-e** [comment]
let's hide the "-" icon if the user hasn't uploaded an image

**Pull request merged**